Doppler.Sap/Services/BillingServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,37 @@
+using Doppler.Sap.Models;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Doppler.Sap.Services
+{
+    public static class ServicesCollectionExtensions
+    {
+        public static void AddServices(this IServiceCollection services, IConfiguration configuration)
+        {
+            services.AddSingleton<DummyBillingService>();
+            services.AddTransient<BillingService>();
+            services.AddSingleton<DummyBusinessPartnerService>();
+            services.AddTransient<BusinessPartnerService>();
+
+            var section = configuration.GetSection("SapConfig");
+            var sapSettings = new SapConfig();
+            section.Bind(sapSettings);
```

Comment:
Was not this configuration section already binded using the `IOptions<>` pattern?